graphettes/Graph.cpp

```diff
@@ -497,7 +497,7 @@ void Graph::rewireRandomEdges(double rewiredEdgesProportion) {
 }
 
 string Graph::autogenFilesFolder() {
-    return "networks/"+name+"/autogenerated/";
+    return "autogenerated/" + name + "/";
```

Comment:
This directory should be defined once, probably at compile time as a #define macro, or in the constructor. Right now you have it hard-coded at least twice: here, and up on line 291. Now you need to maintain the same thing in two different places. Bad. Define it in one place, and use the variable (or constant) everywhere else.